packages/ui/.storybook/preview.css

```
@@ -0,0 +1,3 @@
+@import "tailwindcss";
+
+@source "../src/**/*.{ts,tsx}";
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)